pyproject.toml

```
@@ -1,3 +1,33 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+[project]
+name = "rwth_i6_hlt_pytorch_models"
```

I think we did not talk about the name yet. This would be the PIP package name then, right? Ideally the PIP package name should be the same as the import name. So in setups, you would do `import rwth_i6_hlt_pytorch_models`.

I think the name is too complicated. I would suggest to just use `i6_models` here, which is also not taken by any other project on PyPI yet. Then we can stay with the original `import i6_models`.

```suggestion
name = "i6_models"
```

---

yes I agree